buildSrc/.gitignore

```diff
@@ -0,0 +1 @@
+build-bootstrap/
```

Comment:
It's not a valid dir in the other projects? I dunno, I'm open to moving it there if you think it is that much cleaner.